docs/unified-test-documentation/dasharo-security/200-tpm-support.md

```diff
@@ -404,3 +404,374 @@ Example output:
 ```text
 SpecVersion=2.0
 ```
+
+## TPM003.001 PCRREAD Function (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to verify that PCRREAD function works properly
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `OPERATING_SYSTEM` = Ubuntu 22.04
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+1. Install `tpm2-tools` using:
+
+    ```bash
+    sudo apt-get install tpm2-tools
+    ```
+
+    [Building from source](https://tpm2-tools.readthedocs.io/en/latest/INSTALL/)
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. Execute command in the terminal:
+
+    ```bash
+    sudo tpm2_pcrread
+    ```
+
+1. Note the result
+
+**Expected result**
+
+1. Output from the command shoul look like below:
```

Comment:
`Should look like below`? I will ask in detail and in a clear way what are the conditions for passing this test.

Reply:
Added explanations